### DISCUSSION TITLE
I have some two questions about `tvg` tag

### DISCUSSION THREAD

**Discussion by TechComet** [iptv-org/community#199] [Q&A] [answered]
Hi,

```
"tvg": {
      "id": "cnn.us",
      "name": "CNN",
      "url": "http://epg.streamstv.me/epg/guide-usa.xml.gz"
    }
```

What does `tvg` mean?

`url` what this ?

**Comment by Sphinxroot** [comment, marked as answer]
TV Guide  and url to EPG  (Electronic  Program Guide)  , this will give you TV Guide if your player support it  , as similar to cable Provider
